include/abb_librws/rws_client.h

```diff
@@ -385,6 +385,24 @@ class RWSClient : public POCOClient
    */
   RWSResult getIOSignal(const std::string& iosignal);
 
+  /**
+   * \brief A method for retrieving static information about a mechanical unit.
+   *
+   * \param mechunit for the mechanical unit's name.
```

Comment:
```suggestion
   * \param mechunit name of the mechanical unit.
```

Reply:
Hm... I would prefer to do these kinds of changes in a follow-up PR, since it "brakes" naming consistency with other methods (that are unrelated to this PR).

For example the [getMechanicalUnitJointTarget(...)](https://github.com/jontje/abb_librws/blob/get_mech_unit_info/include/abb_librws/rws_client.h#L407-L413) and [getMechanicalUnitRobTarget(...)](https://github.com/jontje/abb_librws/blob/get_mech_unit_info/include/abb_librws/rws_client.h#L415-L428) methods.

Reply:
"In another PR" -> we have a tendency then never to submit those PRs ;)

Reply:
Well... you shouldn't say never 😅